index.js

```diff
@@ -22,6 +22,8 @@ module.exports = function (opts, callback) {
 
   const images = [].concat(opts.image || [])
   const cwd = path.resolve(opts.cwd || '.')
+  const nodeModulesPath = opts.nodemodulespath ? path.resolve(opts.nodemodulespath) : null;
```

Comment:
Let's resolve it relative to `cwd` (if `nodemodulespath` / `modules` is absolute, that still works):

```suggestion
  const nodeModulesPath = opts.nodemodulespath ? path.resolve(cwd, opts.nodemodulespath) : null;
```